tests/test_dask_image/test_ndfilters/test__conv.py

```diff
@@ -154,3 +154,35 @@ def test_convolutions_compare(sp_func,
             d, weights, origin=origin
         )
     )
+@pytest.mark.parametrize(
```

Comment:
I love how comprehensive this test is with pytest parameterize. Unfortunately this test passes *both* before and after your change, so it's not a good test. But this is easy to fix - in the original bug report we only saw the bug for 1D arrays, so you should choose a 1D array example here as well.